packages/api-bindings/src/graphql/module.graphql

```diff
@@ -0,0 +1,39 @@
+fragment ModuleInfo on ModuleInfo {
+  __typename
+  name
+  type
+}
+
+fragment EnabledModule on EnabledModule {
+  __typename
+  moduleName
+  contractAddress
+  inputParams {
+    ...ModuleInfo
+  }
+  redeemParams {
+    ...ModuleInfo
+  }
+  returnDataParms {
```

Comment:
This is probably a typo in the graphql schema

Reply:
We can fix this in the SDK by aliasing it with the correct one.

Reply:
true